File: web/src/assets/scripts/Bullet.js
export class Bullet {
    constructor(x, y, d, speed, id) {
        this.x = x;
        this.y = y;
        this.c = Math.floor(x);
        this.r = Math.floor(y);
        this.speed = speed;
        this.id = id;

        this.direction = d;
        this.status = "alive";  // alive, dead
    }
}

File: kob/src/main/java/com/phanhom/kob/consumer/Bullet.java
package com.phanhom.kob.consumer;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Bullet {
    private Integer id;
    private double x, y;
    private double speed;
    private Integer direction;
    private String status; // false 代表死亡 true 代表存活
}
